nbconvert/filters/filter_links.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+"""A pandoc filter used in converting notebooks to Latex.
+Converts links between notebooks to Latex cross-references.
+"""
+import re
+
+from pandocfilters import RawInline
+from ..utils.pandoc import applyJSONFilters
+
+def wrapped_convert_link(source):
+    return applyJSONFilters([convert_link], source)
+
+def convert_link(key, val, fmt, meta):
+    """
+    """
+    
+    if key == 'Link':
+        target = val[2][0]
+        # Links to other notebooks
+        m = re.match(r'(\d+\-.+)\.ipynb$', target)
+        if m:
+            return RawInline('tex', 'Section \\ref{sec:%s}' % m.group(1))
+            
+        # Links to sections of this or other notebooks
+        m = re.match(r'(\d+\-.+\.ipynb)?#(.+)$', target)
```

Comment:
Ditto here, the first part of this only makes sense for bookbook joining multiple notebooks together.